test/unit-tests/closeBrowser.test.js

```diff
@@ -9,10 +9,13 @@ describe('close browser successfully',()=>{
     },10000);
     
 
-    test('closeBrowser should return \'Browser Opened\' message',  ()=>{
+    test('closeBrowser should return \'Browser Opened\' message', async ()=>{
 
-        return closeBrowser().then(data => {
-            expect(data).toEqual({ 'description': 'Browser closed' });
+        emitter.on('success', (desc) => {
+            expect(desc).toEqual('Browser opened');
```

Comment:
guess assertion is not waiting for listener here, else it should have failed for `Browser opened` which is expected on `closeBrowser`